istioctl/pkg/tag/generate.go

```diff
@@ -214,9 +221,22 @@ base:
 	return whBuf.String(), nil
 }
 
+func mergeMaps(m1, m2 map[string]string) map[string]string {
+	if m1 == nil {
+		return m2
+	}
+	if m2 == nil {
+		return m1
+	}
+	for k, v := range m2 {
+		m1[k] = v
```

Comment:
does precedence matter here? if there is overlap between m1 and m2, m2 overrides m1.  Or maybe just renaming the function inputs to something like `base` and `override` to make it clear?

Reply:
Thanks, revised. `base` and `override` are much better.